deployment/updates/scripts/xml/notifications/entryClipJobCompletedHttpNotificationTemplate.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+    <config>
+        <serviceUrl>@SERVICE_URL@</serviceUrl>
+        <partnerId>-2</partnerId>
+        <clientTag>Stand alone php 1.0.0</clientTag>
+        <curlTimeout>30</curlTimeout>
+        <userAgent>Stand alone php 1.0.0</userAgent>
+    </config>
+    <session>
+        <partnerId>-2</partnerId>
+        <secret>@ADMIN_CONSOLE_PARTNER_ADMIN_SECRET@</secret>
+        <sessionType>2</sessionType>
+    </session>
+    <multirequest>
+        <request service="eventNotificationTemplate" action="add" plugin="eventNotification" partnerId="0">
+            <template objectType="KalturaHttpNotificationTemplate">
+                <name>Entry Clip Job Finished</name>
+                <systemName>HTTP_ENTRY_CLIP_JOB_FINISHED</systemName>
+                <description>HTTP notification template to be sent when a clip job finished</description>
+                <automaticDispatchEnabled>1</automaticDispatchEnabled>
+                <eventType>7</eventType> <!-- EventNotificationEventType::OBJECT_DELETED -->
+                <eventObjectType>1</eventObjectType> <!-- EventNotificationEventObjectType::ENTRY -->
+                <format>1</format>
+                <eventConditions objectType="array">
+                    <item objectType="KalturaEventFieldCondition">
+                        <description>A replaced entry exists</description>
+                        <field objectType="KalturaEvalBooleanField">
+                            <code>!empty($scope->getObject()->getReplacedEntryId())</code>
+                        </field>
+                    </item>
+                    <item objectType="KalturaFieldMatchCondition">
+                        <description>The replacement is with flowType trim</description>
```

Comment:
@TalDubovKaltura please change description to "The replacing entry flowType is set to 2 (TRIM_CONCAT)"